docs/start.rst

```diff
@@ -13,7 +13,12 @@ To install the Python MOS Interface use::
 Authentication
 --------------
 
-Authentication token.
+An MOS account is required to obtain an authentication token. 
+
+The MOS authentication token may be obtained by logging in to the `MOS Frontend <https://mos.fuinn.ie>`_ and clicking the username in the top right corner.
+
+The token is passed through the ``Interface()`` function documented in the :ref:`ref`. The token may be explicitly passed through in the Python function, or as is implicit in the example below, set as an environment variable.
```

Comment:
I would not call it function but instead refer to it as Interface object: When creating an Interface object, the token may be passed as an argument, otherwise it is assumed to be stored in an environment variable.

Reply:
Thanks, much clearer